src/api.js

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios';
+
+const baseURL = 'https://api.github.com/';
+
+export const getRepos = (query) => {
+  return axios.get(`${baseURL}search/repositories`, {
+    params: {
+      q: query
+    }
+  })
+  .then(res => res.data.items
+    .map(({ id, name, description, html_url }) => ({id, name,description, html_url })
```

Comment:
i choose this solution cause i was helped by @gabro, was actually for learning purpose. I can refactor this map() if you think so. Which lodash function are you thinking about?